pkg/varnishcontroller/varnishadm/varnishadm_test.go

```diff
@@ -350,6 +234,9 @@ func TestGetActiveConfigurationName(t *testing.T) {
 		{
 			nil,
 			func(name string, args ...string) executor {
+				if args[len(args)-1] == "-j" {
+					return &mockExecutor{err: errors.New("err"), response: []byte("Command failed with error code 102\nJSON unimplemented")}
+				}
```

Comment:
So this statement is for backwards compatibility, right?

Reply:
yes